### PR TITLE
Create apps in the working dir if no other dir can be found

### LINKED ISSUE #1386: apps should be created inside the working dir
OSX 10.9.4
Steps to reproduce:
1. Install BRJS in directory $BRJS_HOME
2. Create directory $APP_HOME and open command prompt 
3. Execute command $BRJS_HOME/sdk/brjs create-app wibble
4. App is created in $BRJS_HOME/sdk/brjs-apps/wibble

Should be created in directory I ran command from.

### PR DESCRIPTION
**Pull request by andyberry88**
If `brjs-apps` doesnt exist and `brjs` isnt being run from inside an app use the working dir as the apps directory. Fixes #1386.

**This PR is dependant on #1396 and should be merged after that**

### CODE REVIEW

File: brjs-core-tests/src/test-integration/java/org/bladerunnerjs/api/memoization/FileObserverTest.java [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+			}
+		};
+		
+		return Arrays.asList(new Object[][]{
+				{"Polling", pollingObserverFactory},
+				{"Watching", watchingObserverFactory}
+			});
+	}
+	
+	public FileObserverTest(String testName, FileObserverFactory fileObserverFactory) {
```
Comment (ioanalianabalas) [inline brjs-core-tests/src/test-integration/java/org/bladerunnerjs/api/memoization/FileObserverTest.java]: It doesn't appear as though the parameter `testName` is being used anywhere.

Comment (andyberry88) [inline brjs-core-tests/src/test-integration/java/org/bladerunnerjs/api/memoization/FileObserverTest.java]: It's used via reflection. The first arg on line 54 and 55 are the name for the parameterized test.

**Comment by ioanalianabalas** [comment]
Looks good :+1:

**Comment by ioanalianabalas** [comment]
Just letting you know I have merged in the `develop` branch since this branch could not be merged automatically and I need to branch of it for #1402.

**Pull request merged**